### DISCUSSION TITLE
OpenBadges.me

### DISCUSSION THREAD

**Discussion by jacywest** [MCLD/greatreadingadventure#1008] [Help!]
Is the only way to issue badges (and therefore make challenges) paying for premium features on Open Badges? The API feature is behind a pay wall, but I could be wrong on that or there's another way to connect it to our GRA site.

**Comment by k7hpn** [comment]
I mentioned it in the [other post](/MCLD/greatreadingadventure/discussions/1006) but you should be able to enable the openbadges.me badge maker integrated directly into the GRA by adjusting the setting in: Mission Control -> gear icon -> Site Management -> Settings tab -> Badges.
